make-direct-dit.mk

```diff
@@ -1,46 +1,18 @@
-BASENAME=$(shell yq -r '.catalog.name' < dabl-meta.yaml)
-VERSION=$(shell yq -r '.catalog.version' < dabl-meta.yaml)
+VERSION=$(shell ddit ditversion)
+DIT_NAME=$(shell ddit targetname)
 
-TAG_NAME=${BASENAME}-v${VERSION}
-NAME=${BASENAME}-${VERSION}
-DAR_NAME=${BASENAME}.dar
+TAG_NAME=v${VERSION}
 
 .PHONY: clean install
 
-all: ${NAME}.dit
+all: ${DIT_NAME}
 
-publish: ${NAME}.dit
+publish: ${DIT_NAME}
 	git tag -f "${TAG_NAME}"
-	ghr -replace "${TAG_NAME}" "${NAME}.dit"
+	ghr -replace "${TAG_NAME}" "${DIT_NAME}"
```

Comment:
These two commands will be packaged into `ddit` too, either later this week or early next.